src/augtool.c

```diff
@@ -481,7 +534,14 @@ static int main_loop(void) {
 
     while(1) {
         if (get_line) {
```

Comment:
Maybe it would be easier to refactor before in fact, so as to make this PR more simple.